### ISSUE TITLE
TUTORIAL PASO A PASO PARA SOULISS CON ESP8266

### ISSUE THREAD

**Issue by ladanez** [souliss/souliss.github.io#6]
Bueno, este tutorial básico espero os sea útiles a los que como yo empezamos en esto del hardware.
Tan solo trataremos de usar, en un principio, el módulo ESP8266-01 que es un modelo con menos pines de uso (GPIO) del mercado.

Agradezco a los que me habéis ayudado a orientarme un poco y espero que este manual y otros esté en desarrollo y mejoras para que sea útil a la comunidad.

HARDWARE: (No especificó los cables usb, ordenador y cableado para las conexiones)

- Módulo ESP8266.
![image02](https://cloud.githubusercontent.com/assets/1657291/9608868/d368f570-50d0-11e5-9c3f-77585d4bbc25.jpg)

- FTDI (para poder conectar el módulo ESP8266 al ordenador).
![image19](https://cloud.githubusercontent.com/assets/1657291/9608919/159d5616-50d1-11e5-90dd-acf8f117beb8.jpg)

- Protoboard.
![image14](https://cloud.githubusercontent.com/assets/1657291/9608930/269d1abe-50d1-11e5-8779-36ad3ff43c2c.jpg)

- Móvil/Tablet con android para la instalación del programa SOULISS.

SOFTWARE:
Para poder introducir el sketch en el módulo es necesario instalar el programa arduino que podeis encontrar aqui.(https://www.arduino.cc/en/Main/Software)

Una vez descargado es necesario realizar una actualización de un módulo para poder “grabar” el proyecto (sketch) sobre el módulo ESP8266.

Para ello una vez instalado el programa Arduino iremos al menú superior Archivo>>Preferencias.

Se abrirá la siguiente ventana:
![image16](https://cloud.githubusercontent.com/assets/1657291/9609021/a1725056-50d1-11e5-8e55-996295d03b96.png)

En el área que se muestra en rojo es necesario escribir lo siguiente, como se puede ver en la imagen: (podéis ver la documentación aqui https://github.com/esp8266/Arduino)

http://arduino.esp8266.com/stable/package_esp8266com_index.json

Una vez hecho este paso, vamos a Herramientas >> Placa (la que este seleccionada) >> Boards Manager.
![image08](https://cloud.githubusercontent.com/assets/1657291/9609049/d177a72e-50d1-11e5-82d0-87641dc406a3.png)

Se mostrará la siguiente ventana y hay que bajar hasta encontrar lo siguiente:
![image04](https://cloud.githubusercontent.com/assets/1657291/9609068/e522360e-50d1-11e5-8fec-47795e02513c.png)

Darle al botón “Install” en mi caso como ya está instalado se muestra “Removed”. Paciencia porque tarda un poco, aunque depende de cada conexión de internet.

Para comprobar que todo ha ido correctamente volved a acceder a  Herramientas >> Placa (la que esté seleccionada) >> Boards Manager. Y en la parte inferior del menú deberá verse la placa ESP8266 Generic. 

Seleccionamos dicha placa y dejamos la configuración por defecto que suele ser:
![image16](https://cloud.githubusercontent.com/assets/1657291/9609152/6bb1a632-50d2-11e5-82a7-f917fa3203ab.png)

En mi caso mi placa ESP8266 es exactamente este modelo de Banggood y sin embargo a cargado correctamente el sketch. Así que si tenéis esta o placas similares no os preocupeis, seguro que os carga con esta configuración.

En este momento, ya tenemos configurado el software necesario para cargar cualquier sketch que queramos sobre el módulo ESP8266.

**Comment by plinioseniore** [comment]
Hi,

thanks for this, if you want to get it published in the website you should follow the guidelines here:
https://github.com/souliss/souliss.github.io/blob/master/README.md

Please complete your tutorial with a reference to a sketch, the way on how have the ESP8266 in Boot Mode (GPIO0 to GND and power cycle) the details of the connection and all the other information required to have a node completely setup.

Thanks.

**Comment by juanpintom** [comment]
Hi Dario!, this is the first attempt to add a new tutorial, Im trying to help on it :P 

Regards

**Comment by shineangelic** [comment]
thank you Juan, I agree this should be included in the wiki

**Comment by juanpintom** [comment]
Yes, and shold be traslated to english too :P

I'll work now to add on Tutorials section

Regards


2015-09-02 16:12 GMT+02:00 shineangelic <notifications@github.com>:

**Comment by plinioseniore** [comment]
Tutorial can be spanish :)

**Comment by juanpintom** [comment]
Yes, I will add in spanish, but will be more usefull if its traslated to
english


2015-09-02 16:24 GMT+02:00 Dario Di Maio <notifications@github.com>:

**Comment by juanpintom** [comment]
Dario can you send me the raw text of this issue? I can't edit this to copy
on:
https://github.com/juanpintom/souliss.github.io-juanpintom/blob/master/_posts/media/2015-07-11-Souliss-con-ESP-paso-a-paso.md

I'll ask to the user too

Regards

2015-09-02 16:25 GMT+02:00 Juan Pinto <juanpintom@gmail.com>:

**Comment by plinioseniore** [comment]
Haven't

**Comment by juanpintom** [comment]
Ok, I'll wait to the user from the spanish forum or I'll copy links from images manually (boring... xD)

Regards.
